1_OCaml/TP/TP1.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    let p2 = (2.0.1.0) in
</pre><p>manque une virgule</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>